partner_contact_gender/tests/test_partner_contact_gender.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestPartnerContactGender(TransactionCase):
+    def test_partner_contact_gender(self):
+        from ..hooks import post_init_hook
```

Comment:
You should put in the setUp a test partner with any of the titles, but without gender, call the hook, and then assert the gender to actually test something.